app/controllers/declaration/NactCodeAddController.scala

```diff
@@ -82,7 +82,7 @@ class NactCodeAddController @Inject()(
     implicit request: JourneyRequest[AnyContent]
   ): Future[Result] =
     MultipleItemsHelper
-      .add(boundForm, cachedData, nactCodeLimit)
+      .add(boundForm, cachedData, nactCodeLimit, "nactCode")
```

Comment:
What do you think about having a field in the Form class with the field name ("nactCode") and using the field instead of hard-coded Strings? At the moment it is typo away from errors that may be difficult to find.